tests/homework2/task2/test_cache.py

```diff
@@ -0,0 +1,218 @@
+from collections import OrderedDict
+from unittest import TestCase
+from homework.homework2.task2.cache import cache_decorator
+
+
+class CacheFunctionWithOneArgumentTestCase(TestCase):
+    def test_negative_size_cache(self):
+        with self.assertRaises(ValueError) as context:
+
+            @cache_decorator(size=-162)
+            def fib(n: int) -> int:
+                if n == 1 or n == 2:
+                    return 1
+                return fib(n - 1) + fib(n - 2)
+
+        self.assertTrue("Size must be a non-negative number." in str(context.exception))
```

Comment:
Could you please add some function that can extract an exception message from context by exception type?